deluge/ui/gtkui/path_combo_chooser.py

```diff
@@ -751,7 +750,7 @@ def popup(self):
         PathChooserPopup.popup(self)
         self.popup_window.grab_focus()
 
-        if not self.treeview.flags() & gtk.HAS_FOCUS:
+        if not (self.treeview.has_focus()):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary parens after 'not' keyword](https://www.codacy.com/app/tbkizle/deluge/file/2628318269/issues/source?bid=3238201&fileBranchId=3238205#l753)